addons/account/models/account_journal_dashboard.py

```diff
@@ -257,178 +261,210 @@ def _get_bar_graph_select_query(self):
             'journal_id': self.id
         })
 
-    def get_journal_dashboard_datas(self):
-        currency = self.currency_id or self.company_id.currency_id
-        number_to_reconcile = number_to_check = last_balance = 0
-        has_at_least_one_statement = False
-        bank_account_balance = nb_lines_bank_account_balance = 0
-        outstanding_pay_account_balance = nb_lines_outstanding_pay_account_balance = 0
-        title = ''
-        number_draft = number_waiting = number_late = to_check_balance = 0
-        sum_draft = sum_waiting = sum_late = 0.0
-        if self.type in ('bank', 'cash'):
-            last_statement = self._get_last_bank_statement(
+    def get_journal_dashboard_data_batched(self):
+        self.env['account.move'].flush_model()
+        self.env['account.move.line'].flush_model()
+        dashboard_data = {}
```

Comment:
```suggestion
        dashboard_data = {}  # mutable dict
```
because it's not that common in odoo